### ISSUE TITLE
Dark Variant

### ISSUE THREAD

**Issue by cassidyjames** [samuelrafo/elementary#4]
Soda has a dark theme variant; a dark version of the elementary theme would be awesome. :) Combined with https://github.com/p-e-w/GTKDarkThemeVariantSetter, it would be a fully dark editor.

**Comment by cassidyjames** [comment]
A screenshot of Scratch with the dark theme for reference:
![screenshot from 2015-04-29 22 34 23](https://cloud.githubusercontent.com/assets/611168/7406416/f1b82e56-eebf-11e4-8c32-20d16bc9f074.png)

**Comment by cassidyjames** [comment]
This time with the sidebar files expanded :)


![screenshot from 2015-04-29 22 36 57](https://cloud.githubusercontent.com/assets/611168/7406476/8ef31d70-eec0-11e4-841d-92fa286bfa31.png)

**Comment by piqus** [comment]
Yay! I have been looking for gtk-dark-theme workaround for a year. Thank you for link @cassidyjames. I need to update my fork (https://github.com/piqus/elementary) to include note about this package.

**Comment by cassidyjames** [comment]
@piqus indeed. :) The CSS used for the elementary (both light and dark) theme is available at http://bazaar.launchpad.net/~elementary-design/egtk/trunk/files/head:/gtk-3.0/ if you want to see how to get the dark variant to match closer to what you see in Scratch. ;)

**Comment by piqus** [comment]
I mentioned in samuelrafo/elementary#2 about my fork [piqus/sublime-elementary](https://github.com/piqus/sublime-elementary) which also have dark variant (some screenshots are available in piqus/sublime-elementary#1).
